.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# NB: this file must be named release_prep.sh because the attestation generation doesn't trust user control.
+# see https://github.com/bazel-contrib/.github/blob/v7.2.3/.github/workflows/release_ruleset.yaml#L33-L45
+set -o errexit -o nounset -o pipefail
+
+# Argument provided by reusable workflow caller, see
+# https://github.com/bazel-contrib/.github/blob/v7.2.3/.github/workflows/release_ruleset.yaml#L104
+TAG=$1
+# HACK during debugging, to allow us to fetch older release artifacts.
```

Comment:
Do you need this to be checked in or did you forget to remove it?

Reply:
can remove